tox.ini

```diff
@@ -90,7 +90,7 @@ commands_pre =
   mypyinstalled: pip install file://{toxinidir}/opentelemetry-api/
 
 commands =
-  test: pytest
+  test: pytest {posargs}
```

Comment:
Well, it is useful when running a test case with the debugger and `tox`, like this: `tox ... -- -s`. Pytest requires the `-s` option or it will fail because the console is captured by the debugger.

Reply:
Removed it from this PR to its own.